docs/table_design/dynamic_partitioning.md

```diff
@@ -35,16 +35,19 @@ PROPERTIES(
+| dynamic_partition.prefix                | No       | The prefix added to the names of dynamic partitions. The default value is `p`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| dynamic_partition.buckets               | No       | The number of buckets per dynamic partition. The default value is the same as the number of buckets determined by the reserved word BUCKETS or automatically set by StarRocks.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| dynamic_partition.history_partition_num | No       | The number of historical partitions created by the dynamic partitions mechanism, with a default value of `0`. When the value is greater than 0, historical partitions are created in advance. From V2.5.2, StarRocks supports this parameter.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/c5d643a2-046c-4fe7-a11e-9406ba9f6624)
-> v, in lowecase